MANIFEST.in

```diff
@@ -1,13 +1,36 @@
 include requirements.txt
 include README.rst
+include CONTRIBUTING.rst
 include LICENSE
 
 include linchpin/linchpin.constants
```

Comment:
@p3ck this is where linchpin.constants is now included. I hope that answers your questions.